README.md

```diff
@@ -12,10 +12,15 @@
 </p>
 
 <p align="center">
-  <a href="docs/README.ja.md">日本語</a> | <a href="docs/README.fa.md">فارسی</a> |
-  <a href="docs/README.cn.md">简体中文</a> | <a href="docs/README.ru.md">Русский</a> |
-  <a href="docs/README.es.md">Español</a> | <a href="docs/README.fr.md">Français</a> |
-  <a href="docs/README.kr.md">한국어</a> | <a href="docs/README.ar.md">العربية</a> |
+  <a href="docs/README.ar.md">العربية</a> |
+  <a href="docs/README.cn.md">简体中文</a> |
+  <a href="docs/README.de.md">Deutsch</a> |
+  <a href="docs/README.es.md">Español</a> |
+  <a href="docs/README.fa.md">فارسی</a> |
+  <a href="docs/README.fr.md">Français</a> |
+  <a href="docs/README.ja.md">日本語</a> |
+  <a href="docs/README.kr.md">한국어</a> |
```

Comment:
Thanks for making the HTML prettier! Alphabetizing the language codes is fine with me, but I wonder if there was a reason for the existing order. :thinking:

Reply:
Most of the time this is the easiest for usage and best for maintenance. 🤓